signer/src/transaction_coordinator.rs

```diff
@@ -390,7 +390,10 @@ where
             return Ok(());
         }
 
-        let signer_public_keys = registry_signer_set_info
+        let signer_public_keys = self
+            .context
+            .state()
+            .registry_signer_set_info()
```

Comment:
This also seems a merge leftover -- is there a reason we don't want to reuse `registry_signer_set_info`?